enhancements/node-tuning/mixed-cpu-node-plugin.md

```diff
@@ -0,0 +1,353 @@
+---
+title: mixed-cpu-node-plugin
+authors:
+  - "@Tal-or"
+reviewers: 
```

Comment:
Does the MCO team need to be represented, too, so they're aware of a new type of config being managed?

Reply:
There's no new type of config. 
It's an extension on the existing PerformanceProfile